### PR TITLE
Feature/pages

### LINKED ISSUE #70: pages section
- [x] list of pages (paginated but this might not be implemented yet)
- [x] add new page (new page in admin)  
- [x] edit page 
- [x] delete page 
- [x] search pages in header (like courses, orders, payments etc, might not be implemented) 

if something is blocked by backend ask @pa-cholek for implementation

### PR DESCRIPTION
**Pull request by qunabu**
Closes #70

### CODE REVIEW

File: src/locales/en-US.ts [hunk 1]
```diff
@@ -206,4 +206,11 @@ export default {
   TutorsPopularityMetric: 'Tutors popularity metric',
   reuse_existing: 'Reuse existing',
   open_new_content_editor: 'Open new content editor',
+  content: 'Content',
+  slug_tooltip: 'SLUG is part of Pages URL ',
+  content_tooltip:
+    'The editor is WYSIWYG and includes formatting tools whilst retaining the ability to write markdown shortcuts inline and output plain Markdown.',
+  pages: 'Pages',
+  new_page: 'Nowa static page',
```
Comment (dawidzebacki) [inline src/locales/en-US.ts:214]: Nowa static page -> New static page

File: src/pages/Pages/form.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+import ProForm, { ProFormText, ProFormSwitch } from '@ant-design/pro-form';
+import ProCard from '@ant-design/pro-card';
+import { page as fetchPage, updatePage, createPage } from '@/services/escola-lms/pages';
+
+import WysiwygMarkdown from '@/components/WysiwygMarkdown';
+import { PageContainer } from '@ant-design/pro-layout';
+
+import { useParams, history } from 'umi';
+import { useCallback } from 'react';
+import { useIntl, FormattedMessage } from 'umi';
```
Comment (dawidzebacki) [inline src/pages/Pages/form.tsx:12]: 2 imports from `umi`

File: src/pages/Pages/index.tsx [hunk 3]
```diff
@@ -1,63 +1,67 @@
...
+import { Button, Tooltip, Popconfirm, message } from 'antd';
+import React, { useRef } from 'react';
+import { useIntl, FormattedMessage, Link } from 'umi';
 import { PageContainer } from '@ant-design/pro-layout';
 import type { ProColumns, ActionType } from '@ant-design/pro-table';
 import ProTable from '@ant-design/pro-table';
 
-import { pages } from '@/services/escola-lms/pages';
+import { pages, deletePage } from '@/services/escola-lms/pages';
 import { DeleteOutlined, EditOutlined } from '@ant-design/icons';
```
Comment (dawidzebacki) [inline src/pages/Pages/index.tsx:10]: 2 imports from `@ant-design/icons`

File: src/pages/Pages/index.tsx [hunk 4]
```diff
@@ -1,63 +1,67 @@
...
+        <TypeButtonDrawer
+          key={'user'}
+          type="EscolaLms\Core\Models\User"
+          type_id={record.author_id}
+        />
+      ),
     },
     {
       hideInSearch: true,
       title: <FormattedMessage id="pages.searchTable.titleOption" defaultMessage="操作" />,
```
Comment (dawidzebacki) [inline src/pages/Pages/index.tsx:56]: let's get rid of chinese default messages where you code something or default messages at all where not necessary

**Review by dawidzebacki — APPROVED** [review]

File: src/pages/Pages/index.tsx (hunk 4 above)
Comment (qunabu) [reply src/pages/Pages/index.tsx:56]:
```suggestion
      title: <FormattedMessage id="pages.searchTable.titleOption" defaultMessage="option" />,
```

File: src/pages/Pages/index.tsx [hunk 5]
```diff
@@ -1,63 +1,67 @@
 import { PlusOutlined } from '@ant-design/icons';
```
Comment (qunabu) [inline src/pages/Pages/index.tsx:1]:
```suggestion
import { PlusOutlined,  DeleteOutlined, EditOutlined } from '@ant-design/icons';
```

File: src/pages/Pages/index.tsx (hunk 3 above)
Comment (qunabu) [inline src/pages/Pages/index.tsx:10]:
```suggestion
```

File: src/locales/en-US.ts (hunk 1 above)
Comment (qunabu) [reply src/locales/en-US.ts:214]:
```suggestion
  new_page: 'New static page',
```

File: src/pages/Pages/form.tsx [hunk 6]
```diff
@@ -0,0 +1,123 @@
+import React, { useMemo, useState, useEffect } from 'react';
+import { message, Spin } from 'antd';
+import ProForm, { ProFormText, ProFormSwitch } from '@ant-design/pro-form';
+import ProCard from '@ant-design/pro-card';
+import { page as fetchPage, updatePage, createPage } from '@/services/escola-lms/pages';
+
+import WysiwygMarkdown from '@/components/WysiwygMarkdown';
+import { PageContainer } from '@ant-design/pro-layout';
+
+import { useParams, history } from 'umi';
```
Comment (qunabu) [inline src/pages/Pages/form.tsx:10]:
```suggestion
import { useParams, history, useIntl, FormattedMessage } from 'umi';
```

File: src/pages/Pages/form.tsx (hunk 2 above)
Comment (qunabu) [inline src/pages/Pages/form.tsx:12]:
```suggestion
```

File: src/pages/Pages/index.tsx (hunk 3 above)
Comment (qunabu) [reply src/pages/Pages/index.tsx:10]:
```suggestion
```

**Pull request merged**